modules/core/src/main/java/org/apache/ignite/internal/client/thin/TcpClientTransactions.java

```diff
@@ -94,7 +94,7 @@ private ClientTransaction txStart0(TransactionConcurrency concurrency, Transacti
                 ProtocolContext protocolCtx = req.clientChannel().protocolCtx();
 
                 if (!protocolCtx.isFeatureSupported(TRANSACTIONS)) {
-                    throw new ClientProtocolError(String.format("Transactions are not supported by the server's " +
+                    throw new ClientProtocolError(String.format("Transactions have not supported by the server's " +
```

Comment:
`are not supported` was correct, `have not supported` is not.

Reply:
I know, but our compatibility tests enforce invalid version. Maybe I need to change tests though....

Reply:
Yes, let's fix tests instead